pyvista/examples/downloads.py

```diff
@@ -1739,6 +1739,30 @@ def download_chest(load=True):  # pragma: no cover
     return _download_and_read('MetaIO/ChestCT-SHORT.mha', load=load)
 
 
+def download_brain_atlas_with_sides(load=True):  # pragma: no cover
+    """Download an image of an averaged brain with right-left label.
```

Comment:
```suggestion
    """Download an image of an averaged brain with a right-left label.
```

Reply:
Thank you for your review @tkoyama010 ! I have just pushed the suggested changes.

Reply:
I am not particularly set on how the function to retrieve the example NIfTI image should be named.
Does `download_brain_atlas_with_sides` make sense?

Reply:
Sounds good to me.

Reply:
Cool, thanks @adeak !